lib/services/apiPlatformService.js

```diff
@@ -136,13 +126,11 @@ module.exports = function (apiPlatformRepository, contextHolder, decompresser,
         apiVersionId, deletedFile);
   }
 
-  function getFileWithHash(filePath) {
-    return fileSystemRepository.getFileHash(filePath)
+  function getFileWithHash(file) {
+    return fileSystemRepository.getFileHash(file.path)
```

Comment:
which function? fileSystemRepository.getFileHash? It returns the hash for a file.path, so I think the name is correct